amplemarket.com.domain-setup.json

```diff
@@ -0,0 +1,57 @@
+{
+  "providerId": "amplemarket.com",
+  "providerName": "Amplemarket",
+  "serviceId": "domain-setup",
+  "serviceName": "Amplemarket Domain Setup",
+  "version": 1,
+  "logoUrl": "https://assets.amplemarket.com/assets/images/amplemarket-a-logo-72x72-blue.svg",
+  "description": "Allows users to easily setup DKIM, DMARC, SPF, and tracking pixel for their domain.",
+  "syncBlock": false,
+  "syncPubKeyDomain": "amplemarket.com",
+  "syncRedirectDomain": "app.amplemarket.com,staging.amplemarket.com",
+  "warnPhishing": true,
+  "hostRequired": false,
+  "records": [
+    {
+      "groupId": "base",
+      "type": "CNAME",
+      "host": "t",
+      "pointsTo": "%pixel_tracker_domain%",
+      "ttl": 3600
+    },
+    {
+      "groupId": "base",
+      "type": "TXT",
+      "host": "_dmarc",
+      "data": "%dmarc_value%",
+      "ttl": 3600
```

Comment:
consider setting `txtConflictMatchingMode` to `All` for DMARC to remove duplicates (not yet supported by all DNS providers though).

Reply:
Thank you @pawel-kow 👍 

Added here: https://github.com/Domain-Connect/Templates/pull/510/commits/c75a0f4ad313befe83344894d25da71b49c08fcd